scripts/opt-alive.sh.in

```diff
@@ -14,6 +14,15 @@ PASSES="-passes= argpromotion deadargelim globalopt hotcoldsplit inline ipconstp
 
 TV="-tv"
 for arg in $@; do
+  # update the -passes argument only.
+  shift
+  if [[ $arg == "-passes="* ]]; then
+    passes=${arg#-passes=}
+    passes_new=`@CMAKE_BINARY_DIR@/../scripts/rewritepass.py @LLVM_BINARY_DIR@/bin/opt $passes`
```

Comment:
for the scripts, you should use CMAKE_SOURCE_DIR to avoid assuming anything about the build dir

Reply:
thanks!